ast/substitute/Substitute.cc

```diff
@@ -128,15 +128,27 @@ template <typename T> class SubstWalk {
 };
 } // namespace
 
-ExpressionPtr Substitute::run(core::MutableContext ctx, const core::NameSubstitution &subst, ExpressionPtr what) {
+ParsedFile Substitute::run(core::MutableContext ctx, const core::NameSubstitution &subst, ParsedFile what) {
     SubstWalk walk(subst);
-    what = TreeMap::apply(ctx, walk, std::move(what));
+    what.tree = TreeMap::apply(ctx, walk, std::move(what.tree));
+
+    auto pkg = what.file.data(ctx).getPackage();
+    if (pkg.exists()) {
+        what.file.data(ctx).setPackage(subst.substitute(pkg));
+    }
```

Comment:
oh wow i would not have thought to update this. did this get caught by our tests?

Reply:
It wasn't, but I was thinking about it after having modified the substitution back in December :)